src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -291,6 +292,57 @@ def get_suggested_assignees(
     return []
 
 
+def get_suspect_commit_block(project: Project, event: GroupEvent):
+    """Build up the suspect commit info for the given event"""
+    author = None
+    commit_id = None
+    commit_link = None
+    pr_link = None
+    pr_date = None
+    committers = None
+    repo_base = None
+
+    try:
+        committers = get_serialized_event_file_committers(project, event)
+    except (Release.DoesNotExist, Commit.DoesNotExist):
+        logger.info("Skipping suspect committers because release does not exist.")
+    except Exception:
+        logger.exception("Could not get suspect committers. Continuing execution.")
+    if not committers:
+        return None
+
+    if committers:
+        committer = committers[0]
+        # TODO show the user's Slack name if their identity is linked
+        author = committer.get("author", {}).get("email")
+        commits = committer.get("commits")
+        if commits:
+            commit_id = commits[0].get("id")
+            pull_request = commits[0].get("pullRequest")
```

Comment:
is the first commit the most recent one?

Reply:
That ... is a good question

Reply:
I poked around a bunch and I _think_ it is, I can't find any issues in our Sentry instance or any tests that hit this same function and have more than 1 committer, so I messed around with some [tests](https://github.com/getsentry/sentry/blob/master/tests/sentry/tasks/test_groupowner.py#L133-L151) and created multiple users that set commits, and it only ever returned the most recent one. There are other places in the code that are written expecting the function to return more than 1 commit, but I'm unsure when that actually happens.